tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -0,0 +1,350 @@
+            "test_duration_minutes": test_duration,
+            "warmup_duration_minutes": warmup_duration,
+        }
+
+        # ProducerSwarm parameters
+        #
+
+        producer_kwargs = {}
+        producer_kwargs['min_record_size'] = 64
+        producer_kwargs['max_record_size'] = 64
+
+        effective_msg_size = producer_kwargs['min_record_size'] + (
+            producer_kwargs['max_record_size'] -
+            producer_kwargs['min_record_size']) // 2
+
+        conn_limit = tier_config.connections_limit - 2 * (
+            producers + consumers * subscriptions)
+        _target_per_node = conn_limit // SWARM_WORKERS
+        _conn_per_node = int(_target_per_node * 1)
+
+        msg_rate_per_node = (1 * MiB) // effective_msg_size
+        messages_per_sec_per_producer = max(
+            msg_rate_per_node // _conn_per_node, 1)
+
+        # single producer runtime
+        # Roughly every 500 connection needs 30 seconds to ramp up
+        target_runtime_s = max(30 * math.ceil(_target_per_node // 500),
+                               60 * (test_duration + warmup_duration))
+        records_per_producer = messages_per_sec_per_producer * target_runtime_s
+
+        self._ctx.logger.warn(
+            f"Producers per node: {_conn_per_node} Messages per producer: {records_per_producer} Message rate: {messages_per_sec_per_producer} msg/s"
+        )
+
+        producer_kwargs[
+            'messages_per_second_per_producer'] = messages_per_sec_per_producer
+
+        benchmark = OpenMessagingBenchmark(
+            self._ctx,
+            self.redpanda,
+            "ACK_ALL_GROUP_LINGER_1MS_IDEM_MAX_IN_FLIGHT",
+            (workload, OMBSampleConfigurations.UNIT_TEST_LATENCY_VALIDATOR),
+            num_workers=OMB_WORKERS,
+            topology="ensemble")
+        benchmark.start()
+
+        # Create topic for swarm workers after OMB to avoid the reset
+        swarm_topic_name = "swarm_topic"
+        try:
+            self.rpk.delete_topic(swarm_topic_name)
+        except:
+            pass
+        finally:
+            self.rpk.create_topic(swarm_topic_name,
+                                  tier_config.partitions_upper_limit -
+                                  self._optimal_partition_count(),
+                                  replicas=3)
+
+        swarm = []
+        for _ in range(0, SWARM_WORKERS):
+            _swarm_node = ProducerSwarm(
+                self._ctx,
+                self.redpanda,
+                topic=swarm_topic_name,
+                producers=_conn_per_node,
+                records_per_producer=records_per_producer,
+                timeout_ms=PRODUCER_TIMEOUT_MS,
+                **producer_kwargs)
+
+            swarm.append(_swarm_node)
+
+        for s in swarm:
+            s.start()
+
+        benchmark_time_min = benchmark.benchmark_time() + 5
+        benchmark.wait(timeout_sec=benchmark_time_min * 60)
+        benchmark.check_succeed()
+
+        for s in swarm:
```

Comment:
What happens on exceptions here? Will the swarm guys be left running?

Reply:
Pretty much, ducktape will try to call the worker's stop method after the test runs to end the process. There is no promise that will succeed either though.

Reply:
Oh I see, so DT is aware of the swarm workers and tries to stop them. That's good enough I think?

Reply:
Yeah, specifically the swarm worker is a child class of a DT "service". DT is aware of what services have been allocated to what nodes and will try to destroy them at the end of each test.